mycore-base/src/main/java/org/mycore/services/fieldquery/MCRQuery.java

```diff
@@ -124,14 +125,16 @@ public void setMaxResults(int maxResults) {
      *            the maximum number of results as String
      */
     public void setMaxResults(String maxResultsString) {
-        if (maxResultsString == null || maxResultsString.length() == 0) {
-            this.maxResults = 0;
-        }
-        try {
-            this.maxResults = Integer.parseInt(maxResultsString);
-        } catch (NumberFormatException e) {
-            LOGGER.warn("The Results maxstring " + maxResultsString + " contains not an integer, 0 as default is set");
+        if (StringUtils.isEmpty(maxResultsString)) {
```

Comment:
Ich finde den StringUtils Aufruf nicht gut, der macht genau das gleiche in der gleichen Anzahl der Zeilen Code, hat aber den Nachteil das man nicht gleich sieht was die Methode macht und es ist eine Abhängigkeit.